--- mod_rpaf-0.8.4/mod_rpaf.c~	2015-09-11 16:21:34.000000000 +0300
+++ mod_rpaf-0.8.4/mod_rpaf.c	2015-09-11 16:21:40.142295508 +0300
@@ -420,6 +420,32 @@
                  RSRC_CONF,
                  "Which header to look for when trying to find the real ip of the client in a proxy setup"
                  ),
+
+	/*
+	 * DEPRECATED
+	 * Backward (0.6) compatible config directives for smooth upgrade.
+	 */
+    AP_INIT_FLAG(
+                 "RPAFEnable",
+                 rpaf_enable,
+                 NULL,
+                 RSRC_CONF,
+                 "Enable mod_rpaf"
+                 ),
+    AP_INIT_ITERATE(
+                 "RPAFproxy_ips",
+                 rpaf_set_proxy_ip,
+                 NULL,
+                 RSRC_CONF,
+                 "IP(s) of Proxy server setting X-Forwarded-For header"
+                 ),
+    AP_INIT_FLAG(
+                 "RPAFsethostname",
+                 rpaf_sethostname,
+                 NULL,
+                 RSRC_CONF,
+                 "Let mod_rpaf set the hostname from the X-Host header and update vhosts"
+                 ),
     { NULL }
 };
 
